docs/team/shijiaao.md

```diff
@@ -24,6 +24,7 @@ Given below are my contributions to the project.
 * **Project management**:
     * Managed releases `v1.3.trial` - `v1.3` (2 releases) on GitHub.
```

Comment:
Shall all of us add v1.4 as well?

Reply:
I think adding v1.4 would be good, since theres quite a number of fine tuning done